.github/workflows/docs.yml

```
@@ -37,13 +37,12 @@ jobs:
           python-version: "3.9"
       - name: Install dependencies
         run: |
+          sudo apt-get install -y graphviz
           python -m pip install --upgrade pip
           pip install -r requirements_dev.txt
```

```suggestion
          pip install -r requirements_dev.txt
          echo "$GITHUB_WORKSPACE" >> $GITHUB_PATH
```

add the workspace folder to PATH